### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by lukepaulkopera** [hak86/android_device_hisi_hi6210sft#1]
Hi,
Do you plan to release something soon ???
Thanks

**Comment by hak86** [comment]
yes, i was planning to release aosp mm but i didn't due wifi bug, I don't understand why isn't working on pure android. the rest of features are working fine..
